src/events/ui/EventsCard.tsx

```diff
@@ -37,84 +39,103 @@ export function EventsCard({
-                boxSize="100px"
-                src={imgSrc}
-                alt="image of the event"
-                marginBottom="5"
-              />
-              <Flex>
-                <Tag size="lg" colorScheme="blue" borderRadius="full">
-                  <Avatar boxSize="7" ml={-1} mr={2} />
-                  <TagLabel>{guests}</TagLabel>
-                </Tag>
-              </Flex>
+                padding=".5rem"
+              >
+                <Avatar boxSize="7" mr="1rem" />
+                <TagLabel>{guests}</TagLabel>
+              </Tag>
             </Flex>
-            <Spacer />
-            <Card width="225px">
-              <CardHeader padding="2.5">
-                <Heading size="md" textAlign="center">
-                  {eventName}
-                </Heading>
-              </CardHeader>
-
-              <CardBody>
-                <Stack divider={<StackDivider />} spacing="2">
-                  <Box>
-                    <Heading size="xs" textTransform="uppercase">
-                      Descripcion
-                    </Heading>
-                    <Text pt="2" fontSize="sm">
-                      {description}.
-                    </Text>
-                  </Box>
-                  <Box>
-                    <Heading size="xs" textTransform="uppercase">
-                      Ubicacion
-                    </Heading>
-                    <Text pt="2" fontSize="sm">
-                      {ubication}.
-                    </Text>
-                  </Box>
-                  <Box>
-                    <Heading size="xs" textTransform="uppercase">
-                      Fecha y hora
-                    </Heading>
-                    <Text pt="2" fontSize="sm">
-                      {date}
-                    </Text>
-                    <Text pt="2" fontSize="sm">
-                      {time}
-                    </Text>
-                  </Box>
-                </Stack>
-              </CardBody>
-            </Card>
           </Flex>
-        </CardBody>
-      </Card>
-      <Card>
-        <CardBody>
-          <CardHeader padding="2.5">
-            <Heading size="md" textAlign="center">
-              Nombre
-            </Heading>
-          </CardHeader>
+          <CardBody padding="0 1rem">
+            <CardHeader padding=".4rem 0 0">
+              <Heading size="lg" textAlign="center">
+                {eventName}
+              </Heading>
+            </CardHeader>
+
+            <Box textAlign="center" textColor="#4F10A8">
+              <Text marginTop=".25rem" fontSize="xs">
+                {date}
+              </Text>
+              <Text fontSize="xs">{time}</Text>
+            </Box>
+            <Box textAlign="center" marginTop="2.5rem">
+              <Heading size="xs">Ubicacion</Heading>
+              <Text marginTop=".25rem" fontSize="sm">
+                {ubication}
+                {ubication.split("")[ubication.length - 1] === "." ? null : "."}
```

Comment:
Convert this into a utility function you could register this inside

`src/core/ui/presenters/withTraillingDot`

Reply:
**extra:** try to create a function for `withTraillingDot`